vortexasdk/api/attribute.py

```diff
@@ -1,15 +1,13 @@
-from dataclasses import dataclass
+from typing import Optional
+from pydantic import BaseModel
+
 
-from vortexasdk.api.serdes import FromDictMixin
 from vortexasdk.api.shared_types import Node
 
 
-@dataclass(frozen=True,)
-class Attribute(Node, FromDictMixin):
+class Attribute(Node, BaseModel):
```

Comment:
`Node` inherits from `BaseModel` already, do we need to inherit from it here as well?

Reply:
Fixed in all places